lib/tools/te_enum.h

```diff
@@ -144,6 +144,47 @@ extern int te_enum_translate(const te_enum_trn trn[],
                              int value, te_bool reverse,
                              int unknown_val);
 
+/**
+ * A mapping between a bit number and a value that corresponds to this bit.
+ *
+ * An array of mappings should end with TE_ENUM_BITMASK_CONV_END.
+ */
+typedef struct te_enum_bitmask_conv {
+    /** Bit number. */
+    unsigned int bit;
+    /** Value that is represented by the bit. */
+    uint64_t val;
```

Comment:
Isn't it always 1 << bit ? Why?

If it is bit to bit mapping, may it should be b_bit and the first one should be a_bit?